src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -535,6 +540,66 @@ public override void ProjectStartedHandler(object sender, ProjectStartedEventArg
                     WriteItems(e, e.Items);
                 }
             }
+
+            //node and project context ids for the TargetFramework_mapping key
+            int node = -1;
+            int project_context_id = -1;
+            if (e.BuildEventContext != null)
+            {
+                node = e.BuildEventContext.NodeId;
+                project_context_id = e.BuildEventContext.ProjectContextId;
+            }
+            //creating the value to be added to the TargetFramework_mapping
+            Dictionary<string, string> propertyOutputs = new Dictionary<string, string>();
+            if (e.BuildEventContext != null && e.Items != null)
+            {
+                foreach (DictionaryEntry item in e.Items)
+                {
+                    ITaskItem itemVal = (ITaskItem)item.Value;
+                    //finding if the outputProperties item has been used
+                    if ((string)item.Key == "outputProperties")
```

Comment:
Because of string interning this would actually do the same thing as the .Equals method because I am not creating two instances, I'm essentially referencing the same instance. I can change it for clarity and consistency with other code though